apps/web/app/[language]/homestead/refinedMaterials/data.ts

```diff
@@ -0,0 +1,128 @@
+export interface ConversionRate {
+  // Amount produced
+  produced: number;
+  // Amount required
+  required: number;
+}
+
+export type RefinedCosts = [ConversionRate, ConversionRate, ConversionRate];
+const standardRate = (base: number): RefinedCosts => [
+  {
+    produced: 1,
+    required: base,
+  },
+  {
+    produced: 1,
+    required: base / 2,
+  },
+  base >= 4
+    ? {
+        produced: 1,
+        required: base,
+      }
+    : {
+        produced: 2,
+        required: base / 2,
+      },
+];
+
+export type RefinedSources = {
+  [id: number]: RefinedCosts;
+};
+
+export const WOOD_ID = 103049;
+export const wood: RefinedSources = {
+  19723: standardRate(20),
+  19726: standardRate(12),
+  19727: standardRate(4),
+  19724: standardRate(4),
+  19722: standardRate(4),
+  19725: standardRate(2),
+};
+
+export const METAL_ID = 102205;
+export const metal: RefinedSources = {
+  19702: standardRate(2),
+  19697: standardRate(8),
+  19700: standardRate(4),
+  19701: standardRate(2),
+  19703: standardRate(20),
+  19698: standardRate(8),
+  19699: [
+    {
+      produced: 2,
+      required: 4,
+    },
+    {
+      produced: 2,
+      required: 2,
+    },
+    {
+      produced: 1,
+      required: 1,
+    },
+  ],
+};
+
+export const FIBER_ID = 102306;
+export const fiber: RefinedSources = {
```

Comment:
Does this already contain the 2 anomalies [mentioned on discord](https://discord.com/channels/384735285197537290/384740380014280714/1286246924395216950)?

Reply:
I forgot, it didn't 😅 

I created this small gist as a hack to get the info from the table in the wiki https://gist.github.com/voliva/d1c16e9107e66f103214fe8fbd9d3ee1, and found that actually potato, the one that seemed most efficient, was the anomaly. It's 8-8-4 instead of 8-4-2. But I couldn't find any other... I asked on discord to get more info.